{{cookiecutter.repo_name}}/AUTHORS.md

```diff
@@ -0,0 +1,19 @@
+# Authors
+
+{{cookiecutter.project_name}} was created by {{cookiecutter.author_name}} in {% now 'utc', '%Y' %}.
+
+
+All contributing authors are listed in this file below.
+The repository history at https://github.com/{{cookiecutter.github_url}}
+and the CHANGELOG show individual code contributions.
+
+## Chronological list of authors
+
+<!--
+The rules for this file:
+  * Authors are sorted chronologically
+  * Please give the name you go by, not your GitHub username
+  * Don't ever delete anything
+-->
+
+- {{cookiecutter.author_name}}
```

Comment:
I do like it when authors.md entries have a year entry, not a requirement but it's worth considering?

Reply:
Yeah I thought about it, but I had no preference either way. Happy to go with general consensus

Reply:
I see, if year is programmatically accessible then having them in the file is my vote.

Reply:
Done :)